ToDoOrNotToDo/app/src/main/java/shuvalov/nikita/todoornottodo/SubListActivity.java

```diff
@@ -0,0 +1,117 @@
+package shuvalov.nikita.todoornottodo;
+
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+
+public class SubListActivity extends AppCompatActivity {
+    private RecyclerView subListRecyView;
+    private LinearLayoutManager linearLayoutManager;
+    private Button addItemButt, backButt, removeButt;
+    private int mPosition;
+    private TextView listNameView;
+    private ArrayList<Errands> errandList;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_sub_list);
+
+        Intent superListIntent = getIntent();
+        mPosition=superListIntent.getIntExtra("Position",-1);
+
+        subListRecyView = (RecyclerView) findViewById(R.id.errandRecycler);
+        linearLayoutManager = new LinearLayoutManager(SubListActivity.this, LinearLayoutManager.VERTICAL, false);
+        subListRecyView.setLayoutManager(linearLayoutManager);
+        listNameView = (TextView)findViewById(R.id.listnameview);
+        listNameView.setText(superListIntent.getStringExtra("ListName"));
+
+        addItemButt = (Button)findViewById(R.id.additem);
+        backButt = (Button)findViewById(R.id.backButt);
+        removeButt = (Button)findViewById(R.id.removebutt);
+
+        final SubListRecyclerAdapter subListRecAdp = new SubListRecyclerAdapter(MasterLister.retriveToDoLists().get(mPosition).getErrandsList(),mPosition);
+
+
+        removeButt.setOnClickListener(new View.OnClickListener() {
+                                          @Override
```

Comment:
Careful with indentation here! This gets pretty hard to read when it's indented more than it needs to be.